scm-webapp/src/main/resources/config/gcache.xml

```diff
@@ -86,6 +86,15 @@
     expireAfterWrite="5400"
   />
 
+  <!--
+  Search cache for branch details
+-->
+  <cache
+    name="sonia.cache.cmd.branch-details"
+    maximumSize="10000"
+    expireAfterAccess="60000"
+  />
```

Comment:
What do you think about this cache? It seems too big for me, but I have no idea what an appropriate size might be.